src/loaders/gtdb/gtdb_genome_attribs_loader.py

```diff
@@ -100,11 +100,7 @@ def _rename_col(df, header_mapper):
          None (updates dataframe in place)
     """
 
-    # default mapper (e.g. {'aa_bb_cc': 'Aa Bb Cc'})
-    # Note: all columns are following 'a_b_c' format in the original GTDB metadata file
-    mapper = {col: ' '.join(list(map(lambda x: x.capitalize(), col.split('_')))) for col in df.columns}
```

Comment:
I am okay not doing this. I was thinking when the attribute table was displayed, the header will be presented as `Genome Name` rather than `genome_name`.  If we store attri name as `genome_name`, will it add more work for the front end?

Reply:
It will, but we have to store the keys without spaces or AQL breaks. We could munge the keys every time when returning from the DB, but IMO we should let the front end decide how to display the data

If the FE devs complain, we can figure something else out - maybe we do something similar to `ranks` for the taxa counts and add an endpoint that returns a header mapper. Let's see how the plain old keys work first, maybe we're fixing a problem that doesn't exist

Reply:
ah right. that makes sense.